app/src/main/kotlin/com/wire/android/ui/calling/OngoingCallScreen.kt

```diff
@@ -132,9 +138,9 @@ private fun CallingControls(
             .fillMaxWidth()
             .padding(0.dp, dimensions().spacing16x, 0.dp, 0.dp)
     ) {
-        MicrophoneButton(ongoingCallState.isMuted, onMuteOrUnMuteCall)
+        MicrophoneButton(callState.isMuted) { toggleMute() }
```

Comment:
just to be safe :P do not think this is going to change anything but could oyu change it to MicrophoneButton(callState.isMuted,toggleMute) ?